gnomad_constraint/utils/constraint.py

```diff
@@ -623,3 +839,22 @@ def split_context_ht(
         gerp=gerp_ht[split_context_ht.locus].S,
     )
     return split_context_ht, annotated_context_ht
+
+
+def find_gerp_percentiles(ht: hl.Table) -> Tuple[float, float]:
```

Comment:
If we are not using this in the pipeline, maybe add a note that we are just including it for reference?